docs/sdks/uid2-sdk-ref-java.md

```diff
@@ -286,6 +286,39 @@ If you're using server-only integration (see [Publisher Integration Guide, Serve
 
    If the user has opted out, this method returns null, indicating that the user's identity should be removed from the session. To confirm optout, you can use the `tokenRefreshResponse.isOptout()` function.
 
+## Usage for Advertisers and Data Providers
+1. Create an instance of IdentityMapClient as an instance variable.
+   ```java
+   final private IdentityMapClient identityMapClient = new IdentityMapClient(UID2_BASE_URL, UID2_API_KEY, UID2_SECRET_KEY);
+   ```
+
+
+2. Call a function that takes users' email addresses or phone numbers as input and generates an IdentityMapResponse object. The following example uses email addresses:
+   ```java
+   IdentityMapResponse identityMapResponse = identityMapClient.generateIdentityMap(IdentityMapInput.fromEmails(Arrays.asList("email1@example.com", "email2@example.com")));
+   ```
+
+
+>Note: Inputs will be hashed by the SDK before being sent. This ensures that raw email addresses and phone numbers do not leave your server.
```

Comment:
is it better to use the ":::important .... :::" tag to make this point stand out?

Reply:
I like the idea of making it stand out, but it's not important in the sense that the user needs to do anything. More that it's a good feature for them to know about

Reply:
ok, will see if Gen has any suggestions

Reply:
My comments: First FYI I have to do a pass on the entire docs set to go to Docusaurus admonitions but waiting until EUID is done. Until then, the fewer the better because it's another change we have to make when porting content to GitHub.
On this one -- Warning is the Docu next level up from Note... and is probably not applicable here. So I think Note is good here.

Couple of other comments:

a) line 296
Call a function that takes users' email addresses or phone numbers as input > Call a function that takes email addresses or phone numbers as input
Reasoning: it's not needed, and the s' is no longer mainstream language so I leave it out if it isn't needed.

b) line 302:
Inputs will be hashed by the SDK before being sent > The SDK hashes input values before sending them
Inputs are hashed by the SDK before being sent

c) Also could we change two empty lines to one? line 295, 301.

Reply:
thanks, made those changes